src/asm/section.c

```diff
@@ -583,8 +583,14 @@ void out_PCRelByte(struct Expression *expr)
 /*
  * Output a binary file
  */
-void out_BinaryFile(char const *s)
+void out_BinaryFile(char const *s, int32_t startPos)
 {
+	if (startPos < 0) {
```

Comment:
Maybe we can instead make it relative to the end of the file if it's negative? What do you think?

Reply:
Yes, could do. If we still want to handle files where we don't know the size then it could complicate it a bit though.

Reply:
Bytes can be read into a circular buffer of the correct size for that scenario. I can understand if you don't want to implement that, though—you can make it an error instead, for the time being.

Reply:
Yes, can save that functionality for another pull request :)